addons/sale_project/views/project_sharing_views.xml

```diff
@@ -21,6 +21,9 @@
             <xpath expr="//field[@name='partner_id']" position="after">
                 <field name="sale_line_id" string="Sales Order Item" attrs="{'invisible': [('partner_id', '=', False)]}" options='{"no_open": True}' readonly="1" context="{'create': False, 'edit': False, 'delete': False}"/>
             </xpath>
+            <xpath expr="//field[@name='child_ids']/tree/field[@name='partner_id']" position="attributes">
+                <attribute name="invisible">context.get('should_not_display_partner')</attribute>
```

Comment:
I think it is equivalent to:
```suggestion
                <attribute name="attrs">{'column_invisible': [('parent.allow_billable', '=', False)]}</attribute>
```
right? 🤔 

Also, don't forget to check if the field already had attrs attibute in project, if yes then you will have copy/paste here to avoid losing the attrs definition in project module.